src/components/videos/VideoItem.jsx

```diff
@@ -1,55 +1,85 @@
 import styled from 'styled-components';
 
 const Container = styled.li`
-  width: 300px;
   min-width: 300px;
-  height: 280px;
-  max-height: 280px;
   background-color: #37474f;
   cursor: pointer;
   outline: none;
   border: none;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: flex-start;
   overflow: hidden;
   color: #fff;
   transition: background-color 0.2s ease-in-out;
 
   &:hover {
     background-color: #4c585f;
   }
-  div {
+
+  p {
+    font-size: 14px;
+  }
+
+  .description {
+    height: 60px;
+    max-height: 60px;
+    overflow: hidden;
+  }
+
+  display: flex;
+  justify-content: flex-start;
+
+  width: ${(props) => (props.display === 'grid' ? '300px' : '100%')};
+  height: ${(props) => (props.display === 'grid' ? '280px' : '100px')};
+  max-height: ${(props) => (props.display === 'grid' ? '280px' : '100px')};
+
+  flex-direction: ${(props) => (props.display === 'grid' ? 'column' : 'row')};
+  align-items: ${(props) => (props.display === 'grid' ? 'center' : 'stretch')};
+  margin-bottom: ${(props) => (props.display === 'grid' ? '0' : '10px')};
+
+  .body {
     padding: 6px;
-    padding-top: 12px;
+    padding-top: ${(props) => (props.display === 'grid' ? '12px' : '0')};
     width: 100%;
     color: #fff;
     text-align: left;
   }
+
   img {
     object-fit: cover;
-    height: 140px;
-    min-height: 140px;
-    width: 100%;
+    height: ${(props) => (props.display === 'grid' ? '140px' : '100px')};
+    min-height: ${(props) => (props.display === 'grid' ? '140px' : '100px')};
+    width: ${(props) => (props.display === 'grid' ? '100%' : '160px')};
   }
-  h2 {
-    font-size: 18px;
-    font-weight: 500;
-    margin-bottom: 5px;
-  }
-  p {
-    font-size: 12px;
+
+  .title {
+    max-height: 48px;
+    overflow: hidden;
+    margin-bottom: 10px;
   }
 `;
 
 const VideoItem = (props) => {
```

Comment:
**Optional**:

This is optional because it is a preference but I find it cleaner to destructure and not repeat `props.{something}` on the rest of the component.

```js
const VideoItem = ({id, title, description, display, image}) => {
```

Reply:
Yes, I used to destructuring the properties in the function parameters, but the PRETTIER formatter puts them one below the other and I didn't like how it looks, so that's why I decided to put only "props" as a function parameter. Likewise, what I sometimes do is destructuring at the beginning of the function. Thanks a lot.